plugins/GSdx/Window/GSSetting.cpp

```diff
@@ -215,6 +221,7 @@ const char* dialog_message(int ID, bool* updateText) {
 		default:
 			if (updateText)
 				*updateText = false;
-			return "";
+			return cvtString("");
 	}
 }
+#undef cvtString
```

Comment:
Is this line necessary?

Reply:
Not necessarily although intellisense didn't complain when I tried to use the define elsewhere which led me to believe that it'll clutter up stuff.

Reply:
Ok. I haven't coded for a while and don't recall what's best. I guess it might be better to leave it there.